docs/tutorial/part-five/index.md

```diff
@@ -142,18 +143,17 @@ export const query = graphql`
 `
 ```
 
-The `console.log(data)` line is highlighted above. It's often helpful when
-creating a new component to console out the data you're getting from the GraphQL query
-so you can explore the data in your browser console while building the UI.
+द `console.log(data)` ओळ वर ठळक केली आहे. हे सहसा उपयुक्त ठरते तेव्हा
+आपण ग्राफिक क्वेरीमधून प्राप्त करीत असलेल्या डेटाचे सांत्वन करण्यासाठी एक नवीन घटक तयार करणे
+जेणेकरून आपण यूआय तयार करताना आपल्या ब्राउझर कन्सोलमधील डेटा एक्सप्लोर करू शकता.
 
-If you visit the new page at `/my-files/` and open up your browser console
-you will see something like:
+आपण नवीन पृष्ठास भेट दिली तर `/my-files/` आणि आपला ब्राउझर कन्सोल उघडा आपल्याला असे काहीतरी दिसेल:
 
 ![data-in-console](data-in-console.png)
 
-The shape of the data matches the shape of the GraphQL query.
+डेटाचा आकार ग्राफिक क्वेरीच्या आकाराशी जुळतो.
```

Comment:
please dont translate GraphQL, it is applicable every where, please update

Reply:
all done